subgraph.yaml

```diff
@@ -71,11 +71,13 @@ dataSources:
       apiVersion: 0.0.5
       language: wasm/assemblyscript
       entities:
-        - Operator
+        - PREOperator
```

Comment:
We renamed the entity to `SimplePREApplication`. Let's update here as well.

Reply:
Good catch! Done.